tests/ha/ha_cluster_init.pm

```diff
@@ -34,9 +66,9 @@ sub run {
     # Note: the default timeout need to be increase because it can takes time to join the cluster
     # Initialize the cluster with diskless or shared storage SBD (default)
     $fencing_opt = '-S' if (get_var('USE_DISKLESS_SBD'));
-    if (script_run "ha-cluster-init -y $fencing_opt $unicast_opt", $join_timeout) {
-        assert_script_run "crm -dR cluster init -y $fencing_opt $unicast_opt", $join_timeout;
-    }
+    cluster_init('ha-cluster-init', $fencing_opt, $unicast_opt, $qdevice_opt);
+
+    cluster_init('crm', $fencing_opt, $unicast_opt, $qdevice_opt) if (!wait_serial("ha-cluster-init-finished-0", $join_timeout));
```

Comment:
WDYT about changing the first parameter of `cluster_init()` so it's more clear that on the second call it's using the debug mode of crm? Also a comment between both calls to `cluster_init()` would make it more clear that the second call is only done if the first one was unsuccessful.

Reply:
Do you mean something like `standard` and `debug` as parameters?
I agree for the additional comment.

Reply:
Was thinking about something like `crm-debug-mode` or similar.